app/src/main/java/com/example/ivanpaulrutale/convergelevelup/model/DeveloperDataMapper.java

```diff
@@ -0,0 +1,216 @@
+package com.example.ivanpaulrutale.convergelevelup.model;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+public class DeveloperDataMapper {
```

Comment:
Do you think you need all these fields. Is it possible to pickonly those you need now or maybe in the next tasks?